runtime/suobject.ts

```diff
@@ -93,13 +94,21 @@ export class SuObject extends SuValue {
     Add(x: any = mandatory()): SuObject {
         maxargs(1, arguments.length);
         this.checkReadonly();
+        return this.runWithModificationCheck(this.addFn, x);
+    }
+
+    private addFn(x: any): SuObject {
```

Comment:
Unless we need addFn, putFn, etc. for other reasons, we can just put them inline (probably in an arrow function), like we would with a block in Suneido. Then you don't need to pass any arguments.